scripts/rstrnt-package

```diff
@@ -51,34 +51,34 @@ _pkg_cmd() {
     opr=$1
     success=1
     try=0
-    while [ $try -lt $retry ]; do
-        $pkg_cmd $pkg_args $opr $package
+    while [ "$try" -lt "$retry" ]; do
+        "$pkg_cmd" "$pkg_args" "$opr" "$package"
```

Comment:
I think these should not be quoted

Reply:
I am not against undoing 'quoting everything'.  My concern is the ability to hack on restraint.  For years, there was resistance to it because it is too fragile.  You fix one small thing and something subtle breaks.  So the small fix is undone and the user has to workaround the restraint limitation.  I am wondering how can we start 'doing the right thing' and fix the fallout properly so the tool becomes less fragile and more responsive to getting proper fixes? 
Maybe the first step is to fix the deployment pipeline so that breakage downtime is reduced (or rollback is supported properly).

Reply:
Actually $pkg_args -> $RSTRNT_PKG_ARGS -> -y --idempotent --allow-inactive    so it appears quoting is necessary for line 55.  I am kinda confused how my testing worked here..  Perhaps it was masked by other issues..